src/ui/component/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,24 @@
+import type { FC } from 'react'
+import classNames from 'classnames'
+import './skeleton.scss'
+
+type SkeletonProps = {
+  width?: number
+  height?: number
+  fontSize?: number
+  variant?: 'text' | 'rectangular' | 'rounded' | 'circular'
```

Comment:
I can't see where `rectangular` is defined 🤔

Reply:
I will now add rectangular class and let text for thee default value